src/Console/CastorFileNotFoundApplication.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Castor\Console;
+
+use Castor\Console\Command\InitCommand;
+use Symfony\Component\Console\Application as SymfonyApplication;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/** @internal */
+class CastorFileNotFoundApplication extends SymfonyApplication
+{
+    public function __construct(
+        private readonly \RuntimeException $e,
+    ) {
+        parent::__construct(Application::NAME, Application::VERSION);
+    }
+
+    public function run(InputInterface $input = null, OutputInterface $output = null): int
+    {
+        $this->add(new InitCommand($this->e));
+        $this->setDefaultCommand('init');
+
+        return parent::run($input, $output);
+    }
+
+    protected function getCommandName(InputInterface $input): ?string
+    {
+        if (!\in_array($input->getFirstArgument(), ['completion', 'list', 'help'], true)) {
```

Comment:
Not a big fan of this hardcoded list. I'm thinking about two alternatives more forward compatible:
- retrieves the default registered commands and get their name;
- remove totally this method override and register an event listener on "command not found event" and force the init command in this case (require to init a dispatcher in the application constructor).

Wdyt?

Reply:
I'm fine with this is. It's really hyper stable

Reply:
Just pushed an alternative implementation for this method anyway :angel: